issue-refactor-ci.md

```diff
@@ -153,7 +153,7 @@ Below is a quick inventory of all workflows in `.github/workflows/` with focus o
   - Consume the artifacts built once by a single upstream “Build Artifacts” job
   - Optionally provision a self-hosted runner via `.github/actions/aws-runner`
 - Main “CI Orchestrator” workflow:
-  - Job A: Build devtools/container + component artifacts (collector + nri-init) and publish artifacts
+  - Job A: Build builder/container + component artifacts (collector + nri-init) and publish artifacts
   - Job B: Fast checks (fmt/clippy/units/helm-lint) in parallel, in container
   - Job C: GH-hosted integrations (short)
   - Job D: Conditional heavy workflows, parameterized by test level
```

Comment:
the "Job A, Job B" etc naming is confusing: these are parts of the workflow, but each of these ("Job A", etc) might be composed of multiple GHA jobs